modules/calc/src/main/java/com/opengamma/strata/calc/marketdata/CalculationMarketData.java

```diff
@@ -22,6 +24,12 @@
  * For example a function might request a USD discounting curve, but the scenario market data can contain
  * multiple curve groups, each with a USD discounting curve.
  * <p>
+ * There are two ways to access the available market data.
+ * The first way is to use the method {@link #getValue(MarketDataKey)} which returns
+ * the data associated with a single key for all scenarios.
+ * The second way is to use the method {@link #scenarios()} or {@link #scenario(int)}
+ * which return all the data associated with a single scenario.
+ * <p>
```

Comment:
Probably worth noting that it's more efficient to use `getValue` and most efficient to use `getScenarioValue`.